includes/admin/tools.php

```diff
@@ -35,11 +35,11 @@ function edd_tools_page() {
 	}
 ?>
 
-    <div class="wrap">
-        <h1><?php _e( 'Tools', 'easy-digital-downloads' ); ?></h1>
+	<div class="wrap">
+		<h1><?php esc_html_e( 'Tools', 'easy-digital-downloads' ); ?></h1>
 		<hr class="wp-header-end">
 
-        <h2 class="nav-tab-wrapper edd-nav-tab-wrapper"><?php
+		<nav class="nav-tab-wrapper edd-nav-tab-wrapper" aria-label="<?php esc_attr_e( 'Secondary menu', 'easy-digital-downloads' ); ?>"><?php
```

Comment:
Inconsequential, but:

Can we move the `<?php` opening tag to a new line.
Then at the bottom where we have `?></nav>` can you split those onto separate lines and make sure the indents line up with the opening tags?

(I know it's not related to your changes, but since we're in the area making some markup tweaks, might as well make these line up a bit nicer.)